SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverDebugContext.cpp

```diff
@@ -349,14 +353,51 @@ void USpatialNetDriverDebugContext::KeepActorOnLocalWorker(AActor* Actor)
 	}
 }
 
+void USpatialNetDriverDebugContext::UpdateServerWorkerData()
+{
+	if (NetDriver->ServerWorkerSystemImpl)
+	{
+		UWorld* World = NetDriver->GetWorld();
+		if (World == nullptr)
+		{
+			return;
+		}
+
+		UGameInstance* GameInstance = World->GetGameInstance();
+		if (GameInstance == nullptr)
+		{
+			return;
+		}
+
+		USpatialServerWorkerSystem* ServerWorkerData = GameInstance->GetSubsystem<USpatialServerWorkerSystem>();
+		if (ensure(ServerWorkerData))
+		{
+			SpatialGDK::LegacyLB_CustomWorkerAssignments Assignments;
+			for (const auto& Delegation : SemanticDelegations)
+			{
+				Assignments.LabelToVirtualWorker.Add(Delegation.Key, Delegation.Value);
```

Comment:
Why? I do not want to move any of these values.

Reply:
No slower than Add, and avoids the pitfall of needing to create temporary variables, although this is only future proofing because at present the types are simple.